src/patternfly/components/Tabs/tabs-item-toggle-icon.hbs

```diff
@@ -0,0 +1,7 @@
+<span class="pf-c-tabs__item-toggle-icon">
+  {{#if tabs-item-toggle-icon--IsExpanded}}
```

Comment:
instead of swapping the icon source, we can use a single icon and handle the expanded state change in CSS (`transform: rotate()`). That allows us to transition the toggle rotation, too. You should be able to find an example of this in expandable section, table, data list, tree view, etc.

Reply:
Also looks like the icon color changes on `:hover` (also `:focus` and `:active`), and expanded. The other components with rotation examples should show this, too, except for tree view - that's updated in an open PR.

Reply:
I see the color change when expanded now, but I'm not seeing the color change on `:hover`, `:focus`, and `:active`. How can you tell it's changing in those states?

Reply:
@wise-king-sullyman you should see it when you hover with your mouse, focus with either the mouse or the keyboard, and `:active` is when you hold down your mouse on it.

Also dev tools have ways to trigger these states. This is chrome - you click the `:hov` button at the top/right of the styles pane. 

<img width="718" alt="Screen Shot 2022-05-27 at 3 26 31 PM" src="https://user-images.githubusercontent.com/35148959/170784931-5c90b429-471c-4a35-8a18-ba8665c55f5e.png">